frontend/src/pages/Tutorials/guidedTour/ProjectTour.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+import { ExampleContainer } from '../tutorialsStyle'
+import { useTranslation } from 'react-i18next'
+import { Button } from '/src/components';
+import { ButtonContainer, TourContent, TourOverlay } from './tourStyles';
+
+interface TourProps {
+    onClose: () => void;
+    steps: Step[]
+}
+
+export interface TourContentProps {
+  isBannerStep: boolean
+  tourStep: number
+}
+
+export interface Step {
+  target: string
+  content: string
+  gifUrl: string | null
```

Comment:
This is the one thing that confused me, you can make it be
```suggestion
  gifUrl?: string
```
instead so that you don't need to pass a null value for every instance (it'll be undefined instead)

Reply:
Ah I knew there was a way to do something like this 😂 I tried to type it as `string?`, forgot you have to do it on the name directly, will update

Reply:
Too much kotlin smh my head 🙄